lib/components/footer.jsx

```diff
@@ -63,20 +64,28 @@ export const variants = (metadata, context, _route, routes) => {
   return combinations
 }
 
+const Wrapper = styled(Box) `
+  background-color: ${({
+    theme
+  }) => { return theme.colors.primary.light }};
+`
+
 export const render = (props) => {
   const toc = props.toc.map((page, index) => {
     const url = `/${page.path.join('/')}`
     return (
       <Box key={index} mb={1}>
-        <Link href={url}>{page.title}</Link>
+        <Link color="#527699" fontSize={13} href={url}>
```

Comment:
Why are these link colors hardcoded?

Reply:
We don't have the theme functionality there yet, so we agreed to do it in a simple way to get the first Landr website ready, and then we would tackle the problem when building the next site.